src/app/malaria-101/activity-2/activity-2.component.scss

```diff
@@ -21,3 +22,18 @@
     border-width: 2px;
     border-style: dashed;
 }
+
+#submit-reset {
+    position: absolute;
+    left: $button-left-margin;
+}
+
+#submit {
```

Comment:
Now that you are using a class for the submit and reset buttons, are these css blocks required